scripts/tests/MPS3.conf

```diff
@@ -0,0 +1,26 @@
+# Diable unused terminals
```

Comment:
I think we don't need changes from `scripts/tests` PR. Let's focus on the all-cluster app itself. Build, run and test as other examples.